### DISCUSSION TITLE
Best hardware to flash firmware

### DISCUSSION THREAD

**Discussion by ghost** [StarLabsLtd/firmware#131] [General] [answered]
I had some troubles flashing coreboot with the ch341a, because the chip on the motherboard is so small and hard to clip. Then a friend recommended me the bus pirate 5 that is coming out soon, what are your thoughts on that. For me that's the most annoying part of the starlabs laptops the very small size of the chip where the AMI or coreboot are installed. It would be great to have a better way to flash them.

**Comment by Sean-StarLabs** [comment, marked as answer]
No idea what it is, but plugging a cables much easier - just need something like this:
https://www.amazon.co.uk/sourcing-map-Converter-Connector-Printer/dp/B09VPKL9FV

**Reply by ghost** [reply]
Is there some guide of how to use that one with the starlabs laptops?
